frontend/src/components/atoms/Button.tsx

```diff
@@ -9,6 +9,7 @@ interface ButtonProps {
   size?: "small" | "medium";
   loading?: boolean;
   [key: string]: any;
+  type?: "button" | "submit" | "reset";
```

Comment:
Removed this because it's not actually needed since it is already included in the `[key: string]: any` prop. Removing it means our Button interface is cleaner since we don't need to include any props that are implicitly included with the traditional `button`. e.g. `onClick` is not in our interface but we can still pass it in as usual